src/components/HeaderNav.jsx

```diff
@@ -10,7 +10,7 @@ const HeaderNav = () => {
                 <li><Link to="#" className="header--nav__link">About</Link></li>
                 <li><Link to="#" className="header--nav__link">Contact</Link></li>
                 <li><Link to="#" className="header--nav__link">Add Credits</Link></li>
-                <li><Link to="/Cart" className="header--nav__link">Cart | 3</Link></li>
+                <li><Link to="/Cart" className="header--nav__link"><span>Cart | 3</span></Link></li>
```

Comment:
I think maybe the `span` would just be on the quantity, not the 'Cart |' text, right?

Reply:
I added it to whole `li` to make the the whole cart link look like a button, can definitely just do the number if we want though. Other option could be to have two `span`s and have the second also highlight the number in the cart "button".

Reply:
Ah, I was just thinking about that number changing dynamically. We'll just leave it as is and can sort it out as we go.